components/SignUpResources/SignUpResources.tsx

```diff
@@ -0,0 +1,78 @@
+import clsx from 'clsx'
+import { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import Checkbox from '../Form/Checkbox'
+import Radio from '../Form/Radio'
+
+interface ICategory {
+  name: string
+  id: number
+  translationKey: string
+}
+
+// TODO - Link this with api call (hardcoded for the moment).
+const CATEGORIES: ICategory[] = [
+  {
+    name: 'Servicii',
+    id: 1,
+    translationKey: 'services',
+  },
+  {
+    name: 'Produse',
+    id: 2,
+    translationKey: 'products',
+  },
+  {
+    name: 'Voluntariat',
+    id: 3,
+    translationKey: 'volunteer',
+  },
+  {
+    name: 'Altele',
+    id: 4,
+    translationKey: 'others',
+  },
+]
+
+// TODO - Add here component accordingly with the selection
+const resourceTypeBuilder = (id: number) => {
+  const dictionary = {
+    1: () => <div>{'Services component'}</div>,
+    2: () => <div>{'Products component'}</div>,
+    3: () => <div>{'Volunteer component'}</div>,
+    4: () => <div>{'Others component'}</div>,
+    default: () => <div>{'Others component'}</div>,
+  }
+  return id ? dictionary[id as keyof typeof dictionary]() : dictionary.default()
+}
+
+const SignUpResources = () => {
+  const { t } = useTranslation()
+  const [selectedResourceId, setSelectedResourceId] = useState(1)
+  return (
+    <div className="flex flex-col">
+      <h3 className="mb-2">{t('signup.resources.offer')} *</h3>
+      {CATEGORIES.map(({ id, translationKey }) => (
+        <Radio
```

Comment:
will need to use Checkboxes here
![Screenshot 2022-03-01 at 17 20 23](https://user-images.githubusercontent.com/6429818/156196279-b35b917c-f2cd-4973-8cfa-bdfa32fc248c.png)